README.rst

```diff
@@ -157,6 +202,961 @@ using the ``gridded_perms_of_length`` method.
+       210: (0, 0), (1, 0), (1, 0)
+       210: (0, 0), (0, 0), (1, 0)
+       210: (0, 0), (0, 0), (0, 0)
+
+Due to the crossing ``01`` obstruction it is clear that all of the underlying
+permutations will be decreasing. Moreover, the transition between the left cell
+and the right cell can be between any of the points. In particular, this says
+there are ``n + 1`` gridded permutations of size ``n`` on this tiling. We
+capture this idea by fusing the two columns into a single column.
+
+.. code:: python
+
+       >>> from tilings.strategies import FusionFactory
+       >>> strategy_generator = FusionFactory()
+       >>> for rule in strategy_generator(tiling):
+       ...     print(rule)
+       fuse columns 0 and 1
+       +-+-+                      +-+
+       |\|\|                   ↣  |\|
+       +-+-+                      +-+
+       \: Av(01)                  \: Av(01)
+       Crossing obstructions:     Assumption 0:
+       01: (0, 0), (1, 0)         can count occurrences of
+                                  0: (0, 0)
+
+We use the symbol ``↣`` instead of ``=`` to remind us that the counts of the
+two sides are definitely not the same.
+Notice, the right-hand side tiling here also now requires that we can count the
+occurrences of ``0: (0, 0)``. If there are ``k`` occurrences of ``0: (0, 0)``
+in a gridded permutation then there will be ``k + 1`` gridded permutations that
+fuse to this gridded permutation. Of course, here occurrences of ``0: (0, 0)``
+is going to be equal to the size of the gridded permutation, but in general,
+the points that need to be counted might not cover the whole tiling. For
+example, the following rule was used within specification to enumerate
+``Av(123)``.
+
+.. code:: python
+
+       >>> tiling = Tiling(
+       ...     [
+       ...         GriddedPerm(Perm((0, 1)), ((0, 0), (0, 0))),
+       ...         GriddedPerm(Perm((0, 1)), ((0, 0), (1, 0))),
+       ...         GriddedPerm(Perm((0, 1)), ((1, 0), (1, 0))),
+       ...         GriddedPerm(Perm((0, 1, 2)), ((0, 0), (2, 0), (2, 0))),
+       ...         GriddedPerm(Perm((0, 1, 2)), ((1, 0), (2, 0), (2, 0))),
+       ...         GriddedPerm(Perm((0, 1, 2)), ((2, 0), (2, 0), (2, 0))),
+       ...     ]
+       ... )
+       >>> for rule in strategy_generator(tiling):
+       ...     print(rule)
+       fuse columns 0 and 1
+       +-+-+-+                         +-+-+
+       |\|\|1|                      ↣  |\|1|
+       +-+-+-+                         +-+-+
+       1: Av(012)                      1: Av(012)
+       \: Av(01)                       \: Av(01)
+       Crossing obstructions:          Crossing obstructions:
+       01: (0, 0), (1, 0)              012: (0, 0), (1, 0), (1, 0)
+       012: (0, 0), (2, 0), (2, 0)     Assumption 0:
+       012: (1, 0), (2, 0), (2, 0)     can count occurrences of
+                                       0: (0, 0)
+
+=========
+
+Finally, I'd like reiterate, if you need support, have a suggestion, or just
```

Comment:
Missing "to". Also, perhaps say "we would like to reiterate..." I think "we" has always been used above.

Reply:
Fixed.